corporate/finance/bookkeeping-standards.rst

```diff
@@ -0,0 +1,53 @@
+:og:description: RoboJackets financial records should comply with Generally Accepted Accounting Principles and Internal Revenue Service standards by default. This page documents standards for specific transactions.
+
+Bookkeeping standards
+=====================
+
+.. vale Google.Parens = NO
+.. vale Google.Passive = NO
+.. vale Google.Will = NO
+.. vale proselint.Typography = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+.. vale write-good.TooWordy = NO
+
+RoboJackets financial records should comply with Generally Accepted Accounting Principles and Internal Revenue Service standards by default.
+This page documents standards for specific transactions.
+
+Restricted funds
+----------------
+
+RoboJackets, Inc. generally doesn't accept donations.
+As a 501(c)(7) exempt organization, donations aren't tax-deductible for donors.
+Most donors should be directed to Georgia Tech Foundation, Inc. or another Georgia Tech affiliate to process donations.
```

Comment:
"If someone wants to donate to RoboJackets, they should be directed to the Georgia Tech Foundation"

Reply:
Can you add a suggestion in GitHub, I'm not sure what you're getting at here

Reply:
I think "If someone wants to donate to RoboJackets, they should be directed to the Georgia Tech Foundation" is a better phrasing than "Most donors should be directed to Georgia Tech Foundation, Inc. or another Georgia Tech affiliate to process donations."

Reply:
Pushed a different phrasing